data/templates/set_interfaces.py

```diff
@@ -0,0 +1,34 @@
+import os
+import time
+count = 0
+string= ""
+interfaces=[]
+for dirname, dirnames, filenames in os.walk('/sys/class/net/'):
+    for subdirname in dirnames:
+        if subdirname != "lo":
+            interfaces.append(subdirname)
+            ip= 100+count
+            string += "auto " + subdirname
+            string += "\niface " +  subdirname +" inet static"
+            string += "\naddress 10.0.0." + str(ip)
```

Comment:
@mtannous : please add some comment, so other developer will be aware of this is not a magic IP address, actually any one is OK.